sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -234,28 +234,30 @@ public DirectConnectionConfig setMaxRequestsPerConnection(int maxRequestsPerConn
     }
 
     /**
-     * Gets the request timeout interval
-     * This represents the timeout interval for requests
+     * Gets the network request timeout interval
+     * This represents the time to wait for response from network peer
      *
-     * Default value is 60 seconds
+     * Default value is 5 seconds
```

Comment:
Why is the default 5 seconds when .NET SDK is set to 10 seconds?

Reply:
I am not exactly sure why we are using 5s. For java SDK 4, we can not change/increase this value to 10s since it could be a break change (request latency, retries etc)

In future, we can change it to 10s to align with .Net SDK.